newhelm/cache_helper.py

```diff
@@ -0,0 +1,87 @@
+# mypy: ignore-errors
+from abc import ABC, abstractmethod
+import json
+import os
+from pydantic import BaseModel
+from sqlitedict import SqliteDict  # type: ignore
+from typing import Any, Dict, Generic, Mapping, TypeVar, Optional, get_args
+
+from newhelm.sut import SUT
+from newhelm.typed_data import TypedData
+
+RequestType = TypeVar("RequestType", bound=BaseModel)
+ResponseType = TypeVar("ResponseType", bound=BaseModel)
+
+
+class CacheHelper(ABC, Generic[RequestType, ResponseType]):
```

Comment:
Why have this base? I don't think it is helping you do anything.

Reply:
I was thinking in case we need to make other types of caches (like for Annotators)-- but I think they should be able to use this one as well. I'll remove it.